mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/programmatic/PropertyMappingStep.java

```diff
@@ -44,20 +47,57 @@
 	PropertyMappingDocumentIdOptionsStep documentId();
 
 	/**
+	 * Define a property binder, responsible for creating a bridge.
+	 * To pass some parameters to the bridge,
+	 * use the method {@link #binder(PropertyBinder, Map)} instead.
+	 *
 	 * @param binder A {@link PropertyBinder} responsible for creating a bridge.
 	 * @return {@code this}, for method chaining.
 	 * @see PropertyBinder
 	 */
-	PropertyMappingStep binder(PropertyBinder binder);
+	default PropertyMappingStep binder(PropertyBinder binder) {
+		return binder( binder, Collections.emptyMap() );
+	}
+
+	/**
+	 * Define a property binder, responsible for creating a bridge.
+	 * With this method it is possible to pass a set of parameters to the binder,
+	 * so that they can be used by the bridge.
```

Comment:
```suggestion
	 * With this method it is possible to pass a set of parameters to the binder.
```

Parameters don't have to be used by the bridge, necessarily. They can just control mapping options (is this field sortable, ...).